packages/block-library/src/post-author/index.php

```diff
@@ -117,7 +116,7 @@ function render_block_core_post_author( $attributes, $content, $block ) {
 		array( 'wp-block-post-author' ),
 		isset( $attributes['className'] ) ? array( $attributes['className'] ) : array(),
 		isset( $attributes['itemsJustification'] ) ? array( 'items-justified-' . $attributes['itemsJustification'] ) : array(),
-		isset( $attributes['align'] ) ? array( 'align' . $attributes['align'] ) : array()
+		isset( $attributes['align'] ) ? array( 'has-text-align-' . $attributes['align'] ) : array()
```

Comment:
Is this change related to this PR?

Reply:
That 'align' attribute adjusts the blocks margin on the page.  This attribute corresponds with the text alignment setting so I changed the class name to appropriately reflect that.  

Previously selecting 'align-right' for text alignment would then the block would move to the right of the page, with its text alignment unchanged.

Reply:
So this was a bug.